test/ping-pong/proposer/src/index.mjs

```diff
@@ -3,27 +3,28 @@ Module that runs up as a proposer
 */
 import config from 'config';
 import logger from '../../../../common-files/utils/logger.mjs';
-import Nf3 from '../../../../cli/lib/nf3.mjs';
 import app from './app.mjs';
+import { Nf3Instance } from './nf3-wrapper.mjs';
 
 const environment = config.ENVIRONMENTS[process.env.ENVIRONMENT] || config.ENVIRONMENTS.localhost;
 
-const { mnemonics, signingKeys } = config.TEST_OPTIONS;
-const { PROPOSER_PORT = '', PROPOSER_URL = '' } = process.env;
+const { signingKeys } = config.TEST_OPTIONS;
+const { PROPOSER_PORT } = process.env;
 
 /**
 Does the preliminary setup and starts listening on the websocket
 */
 async function startProposer() {
   logger.info('Starting Proposer...');
-  const nf3 = new Nf3(signingKeys.proposer1, environment);
-  await nf3.init(mnemonics.proposer);
-
+  const nf3 = Nf3Instance(signingKeys.proposer1, environment);
```

Comment:
If `Nf3Instance` is a wrapper around SDK calls, then we should be consistent and use this for the following lines too
- `await nf3.init(undefined, 'optimist');`
- `await nf3.registerProposer(environment.proposerBaseUrl);`

If not, we can remove Nf3Instance wrapper entirely and use `nf3` from `const nf3 = new Nf3(signingKeys.proposer1, environment);` for all the calls to SDK too.

Reply:
Done